ArduPlane/AP_Arming.cpp

```diff
@@ -411,3 +426,20 @@ bool AP_Arming_Plane::mission_checks(bool report)
 #endif
     return ret;
 }
+
+bool AP_Arming_Plane::rc_received_check(bool display_failure)
+{
+    if (rc().enabled_protocols() == 0) {
+        // No protocols enabled, will never get RC, don't block arming
+        return true;
```

Comment:
this is badly named function if it returns true for "rc_received_check()" when there is no RC

Reply:
Changed to `rc_received_if_enabled_check`